docs/Getting Started/Example Apps/index.md

```diff
@@ -4,7 +4,7 @@ SDL provides two example apps: one written in Objective-C and one in Swift. Both
 
 The example apps are located in the [sdl_ios](https://github.com/smartdevicelink/sdl_ios) repository. To try them, you can download the repository and run the example app targets, or you may use `pod try SmartDeviceLink` with [CocoaPods](https://cocoapods.org) installed on your Mac.
 
-!!! NOTE
+!!! IMPORTANT
```

Comment:
Why was this changed?

Reply:
It's important. Most people seem to ignore it when reading this section.